packages/odig/odig.0.0.3/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: ["Daniel Bünzli <daniel.buenzl i@erratique.ch>"]
+homepage: "http://erratique.ch/software/odig"
+doc: "http://erratique.ch/software/odig/doc"
+license: "ISC"
+dev-repo: "http://erratique.ch/repos/odig.git"
+bug-reports: "https://github.com/dbuenzli/odig/issues"
+tags: [ "org:erratique" "build" "dev" "meta" "doc" "packaging" ]
+available: [ ocaml-version >= "4.03"]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
+  "base-unix"
+  "rresult" {>= "0.5.0"}
+  "asetmap"
+  "fpath"
+  "fmt"
+  "logs"
+  "bos" {>= "1.6.0"}
```

Comment:
ping @dbuenzli
There is currently no `"bos" {>= "1.6.0"}`, perhaps  `"bos" {>= "0.1.6"}`?

Reply:
Ha camelus was right after all... I thought the instability check was because of the above warning... thanks for the notification.

Reply:
See https://github.com/ocaml/opam-repository/pull/10579